.github/actions/check-merge-conflict/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Check For Merge Conflict
+description: Checks for merge conflict in Pull Request
```

Comment:
```suggestion
description: Checks for merge conflicts in in the current pull request.
```

Reply:
Next time, please apply these suggestions directly, so the reviewer doesn't have to eyeball them.